CMakeLists.txt

```diff
@@ -103,7 +127,12 @@ CHECK_INCLUDE_FILES("sys/types.h" HAVE_SYS_TYPES_H)
 CHECK_INCLUDE_FILES("sys/vfs.h" HAVE_SYS_VFS_H)
 CHECK_INCLUDE_FILES("sys/prctl.h" HAVE_SYS_PRCTL_H)
 CHECK_INCLUDE_FILES("execinfo.h" HAVE_EXECINFO_H)
-CHECK_INCLUDE_FILES("sched.h" HAVE_SCHED)
+if(LINUX)
+  CHECK_INCLUDE_FILES("sched.h" HAVE_SCHED)
+endif(LINUX)
+if(FREEBSD)
+  CHECK_INCLUDE_FILES("sched.h" HAVE_SCHED_FREEBSD)
```

Comment:
On 20-7-2016 14:25, Kefu Chai wrote:
> In CMakeLists.txt
> <https://github.com/ceph/ceph/pull/10352#discussion_r71514349>:
> 
>> @@ -103,7 +127,12 @@ CHECK_INCLUDE_FILES("sys/types.h" HAVE_SYS_TYPES_H)
>>  CHECK_INCLUDE_FILES("sys/vfs.h" HAVE_SYS_VFS_H)
>>  CHECK_INCLUDE_FILES("sys/prctl.h" HAVE_SYS_PRCTL_H)
>>  CHECK_INCLUDE_FILES("execinfo.h" HAVE_EXECINFO_H)
>> -CHECK_INCLUDE_FILES("sched.h" HAVE_SCHED)
>> +if(LINUX)
>> +  CHECK_INCLUDE_FILES("sched.h" HAVE_SCHED)
>> +endif(LINUX)
>> +if(FREEBSD)
>> +  CHECK_INCLUDE_FILES("sched.h" HAVE_SCHED_FREEBSD)
> 
> okay, can we remove this code block and leave a note here?

It now is for most platforms nothing more that a note.
I'll take the full thing out, and make a note in my todo's that this
should be addressed at one time.